src/routes/voteOnSponsorTime.js

```diff
@@ -86,7 +86,11 @@ module.exports = async function voteOnSponsorTime(req, res) {
       // Send discord message
       if (type != 1) {
           // Get video ID
-          let submissionInfoRow = db.prepare("SELECT videoID, userID, startTime, endTime FROM sponsorTimes WHERE UUID = ?").get(UUID);
+          let submissionInfoRow = db.prepare("SELECT s.videoID, s.userID, s.startTime, s.endTime, u.userName, "+
+            "(select count(1) from sponsorTimes where userID = s.userID) count, "+
+            "(select count(1) from sponsorTimes where userID = s.userID and votes <= -2) disregarded "+
```

Comment:
Instead of 'disregarded", maybe use "ignored" instead?

Reply:
will do

Reply:
Changed it on line 117